smithy-swift-codegen/src/test/kotlin/PaginatorGeneratorTest.kt

```diff
@@ -116,8 +116,8 @@ class PaginatorGeneratorTest {
         /// to access the nested member `[Swift.String:Swift.Int]`
         /// - Returns: `[Swift.String:Swift.Int]`
         extension PaginatorSequence where Input == PaginatedMapInput, Output == PaginatedMapOutputResponse {
-            func mapItems() async throws -> [Swift.String:Swift.Int] {
-                return try await self.asyncCompactMap { item in item.inner?.mapItems }
+            public func mapItems() async throws -> [(String, Swift.Int)] {
+                return try await self.asyncCompactMap { item in item.inner?.mapItems?.map { (${'$'}0, ${'$'}1) } }
```

Comment:
This is expected behavior and actually it is not breaking change because it was not a public API before. The way it works is, when using automatic pagination, customers can target a paginable property to loop through, but when paginable property is map, code breaks. Ideally one should be able to iterate over a map on (key, value) pair which this PR does.

Good call on docs.